app-text/dblatex/files/dblatex-0.3.7-path-logging.patch

```diff
@@ -0,0 +1,18 @@
+# from the wagnerflo overlay, see
+# https://bugs.gentoo.org/show_bug.cgi?id=129368
```

Comment:
Alright, I opened https://sourceforge.net/p/dblatex/patches/9/.  Feel free to comment if you have anything to add.